### ISSUE TITLE
How to Validate AccessToken in WebAPI Core2.0 with IdentityServer3

### ISSUE THREAD

**Issue by UMANGAWASTHY** [OData/WebApi#1662]
Hi I  amusing ASP.NET CORE WEB API 2.0 project.
and my Identity solution build on IdentityServer3. While validating access token, identityServer3.AccessTokenvalidation not supported in .NET Core 2.0 platform. How to rectify this issue.

**Comment by biaol-odata** [comment]
@UMANGAWASTHY Looks like you are using the general WebAPI 2.0, which is part of https://github.com/aspnet/AspNetCore. 

Authentication/authorization should be done by other middle ware. 
WebAPI-OData here is a plugin specific for supporting OData in WebAPI.

**Comment by UMANGAWASTHY** [comment]
@biaol-odata  yes, i am using WebAPI 2.0, Could you forward some links, demo examples to verify for other middle ware, which you referring too.
